Jetcaster/wear/src/main/java/com/example/jetcaster/ui/library/PodcastsScreen.kt

```diff
@@ -0,0 +1,202 @@
+                it.takeIf { it.title.isNotEmpty() }
+                    ?: it.copy(title = stringResource(id = R.string.no_title))
+            }
+
+            PodcastsScreenState.Loaded(modifiedPodcast)
+        }
+
+        PodcastsScreenState.Failed,
+        PodcastsScreenState.Loading,
+        -> uiState
+    }
+
+    PodcastsScreen(
+        podcastsScreenState = modifiedState,
+        onPodcastsItemClick = onPodcastsItemClick
+    )
+
+    if (modifiedState == PodcastsScreenState.Failed) {
+        Dialog(
+            showDialog = true,
+            onDismissRequest = onErrorDialogCancelClick,
+            scrollState = rememberScalingLazyListState(),
+        ) {
+            Alert(
+                title = {
+                    Text(
+                        text = stringResource(R.string.podcasts_no_podcasts),
+                        color = MaterialTheme.colors.onBackground,
+                        textAlign = TextAlign.Center,
+                        style = MaterialTheme.typography.title3,
+                    )
+                },
+            ) {
+                item {
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                    ) {
+
+                        Button(
+                            imageVector = Icons.Default.Close,
+                            contentDescription = stringResource(
+                                id = R.string
+                                    .podcasts_failed_dialog_cancel_button_content_description,
+                            ),
+                            onClick = onErrorDialogCancelClick,
+                            modifier = Modifier
+                                .size(24.dp)
+                                .wrapContentSize(align = Alignment.Center),
+                            colors = ButtonDefaults.secondaryButtonColors()
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@ExperimentalHorologistApi
+@Composable
+fun PodcastsScreen(
+    podcastsScreenState: PodcastsScreenState<PodcastInfo>,
+    onPodcastsItemClick: (PodcastInfo) -> Unit,
+    modifier: Modifier = Modifier,
+    podcastItemArtworkPlaceholder: Painter? = null,
+) {
+
+    val podcastContent: @Composable (podcast: PodcastInfo) -> Unit = { podcast ->
+        Chip(
+            label = podcast.title,
+            onClick = { onPodcastsItemClick(podcast) },
+            icon = CoilPaintable(podcast.imageUrl, podcastItemArtworkPlaceholder),
+            largeIcon = true,
+            colors = ChipDefaults.secondaryChipColors(),
+        )
+    }
+
+    PodcastsScreen(
+        podcastsScreenState = podcastsScreenState,
+        podcastContent = podcastContent,
+        modifier = modifier,
+    )
+}
+
+@ExperimentalHorologistApi
+@Composable
+fun <T> PodcastsScreen(
+    podcastsScreenState: PodcastsScreenState<T>,
+    podcastContent: @Composable (podcast: T) -> Unit,
```

Comment:
Consider naming this as `content` and make it the last argument this composable accepts so that you can use the trailing lambda syntax at the call site. Our guidance recommends this pattern: https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#compose-ui-layouts 

```suggestion
    content: @Composable (podcast: T) -> Unit,
```

Reply:
thanks